webpack.config.js

```diff
@@ -89,9 +99,11 @@ const config = {
           },
         ],
       },
+      // ESM support. See: https://github.com/apollographql/react-apollo/issues/1737#issuecomment-371178602
       {
-        test: /\.json$/,
-        loader: "json-loader",
+        type: "javascript/auto",
+        test: /\.mjs$/,
+        use: [],
```

Comment:
This is interesting... I've never seen this being required before 🤔

Reply:
Yah I needed it a while back but i'm curious if I still do...